backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json

```diff
@@ -76,6 +76,20 @@
           "sortable": true
         }
       },
+      "Criteria": {
+        "edit": {
+          "label": "Criteria",
+          "description": "",
+          "placeholder": "",
```

Comment:
Can we have some examples as placeholders? You can get examples from the Figma :)

Reply:
placeholder is not a description. Placeholder is to give an example you can put text like this.

ex) For wives, For members age 19+

Reply:
Sorry for my misunderstanding and thank you for the details! I made changes.